lib/src/validator.dart

```diff
@@ -190,4 +202,27 @@ abstract class Validator {
       }
     });
   }
+
+  /// Returns the [files] that are inside [dir] (relative to the package
+  /// entrypoint).
+  List<String> filesBeneath(String dir, {required bool recursive}) {
```

Comment:
Maybe we should place this in a more central place?

Reply:
Let's do that later.
I added a todo.